maven/Release/Jenkinsfile

```diff
@@ -1,31 +1,23 @@
-#!/usr/bin/groovy
+@Library('github.com/fabric8io/osio-pipeline@master') _
 
-@Library('github.com/fabric8io/fabric8-pipeline-library@master')
-def canaryVersion = "1.0.${env.BUILD_NUMBER}"
-def utils = new io.fabric8.Utils()
+osio {
 
-mavenNode {
-  checkout scm
-  if (utils.isCI()) {
+  config runtime: 'java', version: '1.8'
 
-    mavenCI {
-        integrationTestCmd =
-             "mvn org.apache.maven.plugins:maven-failsafe-plugin:integration-test \
-                org.apache.maven.plugins:maven-failsafe-plugin:verify \
-                -Dnamespace.use.current=false -Dnamespace.use.existing=${utils.testNamespace()} \
-                -Dit.test=*IT -DfailIfNoTests=false -DenableImageStreamDetection=true \
-                -P openshift-it"
-    }
+  ci {
 
-  } else if (utils.isCD()) {
-    echo 'NOTE: running pipelines for the first time will take longer as build and base docker images are pulled onto the node'
-    container(name: 'maven', shell:'/bin/bash') {
-      stage('Build Image') {
-        mavenCanaryRelease {
-          version = canaryVersion
-        }
-      }
-    }
+     integrationTestCmd = "mvn verify integration-test -Dnamespace.use.current=false -Dnamespace.use.existing=${testNamespace()} -Dit.test=*IT -DfailIfNoTests=false -DenableImageStreamDetection=true -Popenshift,openshift-it"
+     runTest commands: integrationTestCmd
   }
-}
 
+  cd {
+
+    // processing openshift template present in .openshiftio/application.yaml
+    def resources = processTemplate(params: [
+          release_version: "1.0.${env.BUILD_NUMBER}"
```

Comment:
@rupalibehera could you make it as `RELEASE_VERSION` (all uppercase)

Reply:
done.